README.md

```diff
@@ -215,7 +215,17 @@ Same using pronto, fetching ontology from obolibrary
 runoak -i obolibrary:go.obo  viz GO:0005773
 ```
 
-## Potential Refactoring
+## Developer notes
+### Local project setup
+Prerequisites:
+1. Python 3.10+
```

Comment:
3.9 works

In fact if you look at the checks we do
https://github.com/INCATools/ontology-access-kit/blob/main/.github/workflows/main.yaml

The *only* guarantee we have is for 3.9

Reply:
Thanks! This was just a guess on my part. will change it now